code/modules/vore/eating/bellymodes_vr.dm

```diff
@@ -51,6 +51,7 @@
 			if(M.stat == DEAD)
 				var/digest_alert_owner = pick(digest_messages_owner)
 				var/digest_alert_prey = pick(digest_messages_prey)
+				var/compensation = M.getOxyLoss()
```

Comment:
I had no idea what this compensation thing was about until someone else explained it.  Its weird enough that it requires documenting in comments I'd say.

Reply:
Roger that